### PR TITLE
feat(internal-plugin-metrics): add more verbs to taxonomy for behavioral metrics

### PR DESCRIPTION
**Pull request by lialang-cisco**
# COMPLETES #< INSERT LINK TO ISSUE >

## This pull request addresses https://jira-eng-sjc12.cisco.com/jira/browse/WXCC-4635

< DESCRIBE THE CONTEXT OF THE ISSUE >

## by making the following changes, we allow consumers of internal-plugin-metrics to submit metrics, with several more possible verbs as part of the taxonomy of metric names, `product.agent.target.verb`.

< DESCRIBE YOUR CHANGES >

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdkx/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/internal-plugin-metrics/src/metrics.types.ts [hunk 1]
```diff
@@ -51,7 +51,10 @@ export type MetricEventVerb =
   | 'abort'
   | 'sync'
   | 'login'
-  | 'logout';
+  | 'logout'
+  | 'answer'
+  | 'activate'
+  | 'deactivate';
```
Comment (mkesavan13) [inline packages/@webex/internal-plugin-metrics/src/metrics.types.ts:57]: Should we try and bring in all the WxCC-specific verbs in a separate variable and export the merged list?

**Review by mkesavan13 — APPROVED** [review]
Looks good to me

File: packages/@webex/internal-plugin-metrics/src/metrics.types.ts (hunk 1 above)
Comment (lialang-cisco) [reply packages/@webex/internal-plugin-metrics/src/metrics.types.ts:57]: I think the point is that all these verbs should be common to both the Webex web clinet and WxCC, and any other future consumers, I think we're best to stick with a single list.

Comment (torgeadelin) [reply packages/@webex/internal-plugin-metrics/src/metrics.types.ts:57]: yes, we should stick to 1 list

**Review by torgeadelin — APPROVED** [review]

**Pull request merged**